packages/slice-machine/components/Simulator/components/Header/index.tsx

```diff
@@ -78,15 +57,28 @@ const Header: React.FunctionComponent<PropTypes> = ({
         }}
       >
         <SliceMachineLogo height={"20px"} width={"20px"} />
-        <Text mx={2}>{Model.model.name}</Text>
+        <Text mx={2}>{slice.model.name}</Text>
+        <VarationsPopover
+          defaultValue={variation}
+          variations={slice.model.variations}
+          onChange={(v) => redirect(slice, v, true)}
+          disabled={slice.model.variations.length <= 1}
+        />
+      </Flex>
+      <Flex sx={{ alignItems: "center", justifyContent: "space-between" }}>
+        <Flex sx={{ alignItems: "center", justifyContent: "space-around" }}>
+          <Flex sx={{ alignItems: "center" }}>
+            <Label htmlFor="show-mock-editor">Show the mock editor</Label>
+            <Switch
+              id="show-mock-editor"
+              checked={isDisplayEditor}
+              onChange={toggleIsDisplayEditor}
+            />
+          </Flex>
+        </Flex>
+        <Button onClick={() => console.log("todo")} label="Save mock" />
```

Comment:
Can we ensure that this is not visible on the UI until we actually implement the save action